Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
I can reproduce it in Chrome and Firefox when either letting the animation play or dragging the time slider.  I dunno, it could be a browser/OS/driver issue.

Can we show these CZML features using another approach?